applications/fov-quicklook/templates/coordinator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: quicklook-coordinator
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: quicklook-coordinator
+  template:
+    metadata:
+      labels:
+        app: quicklook-coordinator
+    spec:
+      containers:
+        - name: quicklook-coordinator
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              set -e
+              alembic upgrade head
+              exec python -m quicklook.coordinator.api
+          env:
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: quicklook-db
```

Comment:
I think this is OK for now.  Normally for a phalanx application, this would be still in the application secret (`fov-quicklook`) and there would be a `generate` clause in the `secrets.yaml` for it.

Reply:
fixed in f214bf05e9984fbe8ad82f4186cbbbab4f8e1f93